File: app/src/main/java/com/example/vn008xw/carbeat/data/vo/AbsentLiveData.java
package com.example.vn008xw.carbeat.data.vo;

import android.arch.lifecycle.LiveData;

/**
 * Created by vn008xw on 6/22/17.
 */

public class AbsentLiveData extends LiveData {

  private AbsentLiveData() {
    postValue(null);
  }

  public static <T> LiveData<T> create() {
    return new AbsentLiveData();
  }
}

File: app/src/main/java/com/example/vn008xw/carbeat/data/vo/AbsentLiveData.kt
package com.example.vn008xw.carbeat.data.vo

import android.arch.lifecycle.LiveData

/**
 * Created by vn008xw on 6/18/17.
 */
class AbsentLiveData<T> private constructor() : LiveData<T>() {

    init {
        postValue(null)
    }

    companion object {
        fun <T> create(): LiveData<T> {
            return AbsentLiveData()
        }
    }
}